<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> publications | Charikleia (Clio) Karageorgiou </title> <meta name="author" content="Charikleia (Clio) Karageorgiou"> <meta name="description" content="publications in reversed chronological order"> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%A7%AC&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://biolevol.github.io/publications/"> <script src="/assets/js/theme.js?a81d82887dd692e91686b43de4542f18"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>
    initTheme();
  </script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Charikleia</span> (Clio) Karageorgiou </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">research </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">outreach and service </a> </li> <li class="nav-item "> <a class="nav-link" href="/news/">news </a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description">publications in reversed chronological order</p> </header> <article> <script src="/assets/js/bibsearch.js?1bc438ca9037884cc579601c09afd847" type="module"></script> <p><input type="text" id="bibsearch" spellcheck="false" autocomplete="off" class="search bibsearch-form-input" placeholder="Type to filter"></p> <div class="publications"> <h2 class="bibliography">2025</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Scheer2025-ub" class="col-sm-8"> <div class="title">Rapid Adaptive Increase of Amylase Gene Copy Number in Indigenous Andeans</div> <div class="author"> Kendra Scheer, Luane J B Landau, Kelsey Jorgensen, and <span class="more-authors" title="click to view 14 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '14 more authors' ? 'Charikleia Karageorgiou, L Siao, C Alkan, A M Morales-Rivera, C Osbourne, O Garcia, L Pearson, M Kiyamu, F Leon-Velarde, M Rivera-Chira, F Lee, T Brutsaert, A Bigham, O Gokcumen' : '14 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">14 more authors</span> </div> <div class="periodical"> <em>bioRxiv</em>, Mar 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1101/2025.03.25.644684" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1101/2025.03.25.644684" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>The salivary amylase gene (AMY1) exhibits remarkable copy number variation linked to dietary shifts in human evolution. While global studies highlight its structural complexity and association with starch-rich diets, localized selection patterns remain under explored. Here, we analyzed AMY1 copy number in 3,723 individuals from 85 populations, revealing that Indigenous Peruvian Andean populations possess the highest AMY1 copy number globally. A genome-wide analysis showed significantly higher amylase copy numbers in Peruvian Andean genomes compared to closely related populations. Further, we identified positive selection (selection coefficient of 0.0124, log likelihood ratio of 11.1543) at the nucleotide level on a haplotype harboring at least five haploid AMY1 copies, with a Peruvian Andean-specific expansion coinciding with potato domestication (Â  6-10 kya). Using ultra-long-read sequencing, we demonstrated that previously-described recombination-based mutational mechanisms drive the formation of high-copy AMY1 haplotypes observed in Andean population. Our study provides a framework for investigating structurally complex loci and their role in human dietary adaptation.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Karageorgiou2025-nu" class="col-sm-8"> <div class="title">Convergent evolution through independent rearrangements in the primate amylase locus</div> <div class="author"> <em>Charikleia Karageorgiou</em>, Stefan Ruhl, and Omer Gokcumen </div> <div class="periodical"> <em>bioRxiv</em>, Aug 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1101/2025.08.14.670395" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1101/2025.08.14.670395" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Structurally complex regions of the genome are increasingly recognized as engines of evolutionary convergence due to their propensity to generate recurrent gene duplications that give rise to similar gene expression patterns and traits across lineages. However the mutational mechanisms driving these duplications and the regulatory changes enabling novel expression patterns remain poorly understood. The primate amylase locus, marked by independent gene duplications, provides an ideal model to investigate these dynamics. Leveraging high-quality genome assemblies from 53 primates and multi-tissue transcriptomes from Old World monkeys, we reconstructed the evolutionary history of the recurrent gene duplications across the primate phylogeny. Our data suggest that lineage-specific LTR retrotransposon insertions are associated with initial structural instability, while subsequent duplications are primarily driven by non-allelic homologous recombination. Recurrent independent duplications in rhesus macaques, olive baboons, and great apes gave rise to distinct amylase gene copies with convergent expression in the pancreas and salivary glands. We found that these independent gene duplications are accompanied by episodic diversifying selection on lineage-specific copies, likely driving the emergence of functional divergence. Our comparative analyses in primates indicate that the gene ancestral to great ape AMY1 and AMY2A was expressed in both salivary glands and pancreas in the Catarrhini ancestor. The great ape-specific duplication of this ancestral gene likely facilitated subfunctionalization into salivary gland- and pancreas-specific expression, respectively. Comparative analysis of primate amylase promoter regions reveals regulatory rewiring, driven by motif turnover mediated by structural rearrangements, and partially explaining evolutionary shifts in expression. Together, our findings highlight how structural and regulatory modularity in complex genomic regions drives evolutionary innovation and molecular convergence, and we provide a genomic framework for dissecting these processes across diverse lineages.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Karageorgiou2024-hk" class="col-sm-8"> <div class="title">Deciphering the role of structural variation in human evolution: a functional perspective</div> <div class="author"> <em>Charikleia Karageorgiou</em>, Omer Gokcumen, and Megan Y Dennis </div> <div class="periodical"> <em>Curr. Opin. Genet. Dev.</em>, Oct 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.gde.2024.102240" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.gde.2024.102240" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Advances in sequencing technologies have enabled the comparison of high-quality genomes of diverse primate species, revealing vast amounts of divergence due to structural variation. Given their large size, structural variants (SVs) can simultaneously alter the function and regulation of multiple genes. Studies estimate that collectively more than 3.5% of the genome is divergent in humans versus other great apes, impacting thousands of genes. Functional genomics and gene-editing tools in various model systems recently emerged as an exciting frontier - investigating the wide-ranging impacts of SVs on molecular, cellular, and systems-level phenotypes. This review examines existing research and identifies future directions to broaden our understanding of the functional roles of SVs on phenotypic innovations and diversity impacting uniquely human features, ranging from cognition to metabolic adaptations.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Papadadonakis2024-hq" class="col-sm-8"> <div class="title">Evolution of gene regulatory networks by means of selection and random genetic drift</div> <div class="author"> Stefanos Papadadonakis, Antonios Kioukis, <em>Charikleia Karageorgiou</em>, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Pavlos Pavlidis' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>PeerJ</em>, Aug 2024 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.7717/peerj.17918" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.7717/peerj.17918" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Running2024-sd" class="col-sm-8"> <div class="title">Investigating the mechanism of neurotoxic effects of PFAS in differentiated neuronal cells through transcriptomics and lipidomics analysis</div> <div class="author"> Logan Running, Judith R Cristobal, <em>Charikleia Karageorgiou</em>, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Michelle Camdzic, John Michael N Aguilar, Omer Gokcumen, Diana S Aga, G Ekin Atilla-Gokcumen' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>ACS Chem. Neurosci.</em>, Dec 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1021/acschemneuro.4c00652" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1021/acschemneuro.4c00652" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Per- and polyfluorinated alkyl substances (PFAS) are pervasive environmental contaminants that bioaccumulate in tissues and pose risks to human health. Increasing evidence links PFAS to neurodegenerative and behavioral disorders, yet the underlying mechanisms of their effects on neuronal function remain largely unexplored. In this study, we utilized SH-SY5Y neuroblastoma cells, differentiated into neuronal-like cells, to investigate the impact of six PFAS compoundsâperfluorooctanoic acid (PFOA), perfluorooctanesulfonic acid (PFOS), perfluorodecanoic acid (PFDA), perfluorodecanesulfonic acid (PFDS), 8:2 fluorotelomer sulfonate (8:2 FTS), and 8:2 fluorotelomer alcohol (8:2 FTOH)âon neuronal health. Following a 30 \muM exposure for 24 h, PFAS accumulation ranged from 40-6500 ng/mg of protein. Transcriptomic analysis revealed 721 differentially expressed genes (DEGs) across treatments (padj &lt; 0.05), with 11 DEGs shared among all PFAS exposures, indicating potential biomarkers for neuronal PFAS toxicity. PFOA-treated cells showed downregulation of genes involved in synaptic growth and neural function, while PFOS, PFDS, 8:2 FTS, and 8:2 FTOH exposures resulted in the upregulation of genes related to hypoxia response and amino acid metabolism. Lipidomic profiling further demonstrated significant increases in fatty acid levels with PFDA, PFDS, and 8:2 FTS and depletion of triacylglycerols with 8:2 FTOH treatments. These findings suggest that the neurotoxic effects of PFAS are structurally dependent, offering insights into the molecular processes that may drive PFAS-induced neuronal dysfunction.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Yilmaz2024-xf" class="col-sm-8"> <div class="title">Reconstruction of the human amylase locus reveals ancient duplications seeding modern-day variation</div> <div class="author"> Feyza Yilmaz<sup>*</sup>, <em>Charikleia Karageorgiou<sup>*</sup></em>, Kwondo Kim<sup>*</sup>, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Petar Pajic, Kendra Scheer, Human Genome Structural Variation Consortium, Christine R Beck, Ann-Marie Torregrossa, Charles Lee, Omer Gokcumen' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>Science</em>, Nov 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1126/science.adn0609" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-doi="10.1126/science.adn0609"></span> <span class="__dimensions_badge_embed__" data-doi="10.1126/science.adn0609" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Previous studies suggested that the copy number of the human salivary amylase gene, AMY1, correlates with starch-rich diets. However, evolutionary analyses are hampered by the absence of accurate, sequence-resolved haplotype variation maps. We identified 30 structurally distinct haplotypes at nucleotide resolution among 98 present-day humans, revealing that the coding sequences of AMY1 copies are evolving under negative selection. Genomic analyses of these haplotypes in archaic hominins and ancient human genomes suggest that a common three-copy haplotype, dating as far back as 800,000 years ago, has seeded rapidly evolving rearrangements through recurrent nonallelic homologous recombination. Additionally, haplotypes with more than three AMY1 copies have significantly increased in frequency among European farmers over the past 4000 years, potentially as an adaptive response to increased starch digestion.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Labbe2023-bg" class="col-sm-8"> <div class="title">Genomic analysis of two phlebotomine sand fly vectors of Leishmania from the New and Old World</div> <div class="author"> FrÃ©dÃ©ric LabbÃ©, Maha Abdeladhim, Jenica Abrudan, and <span class="more-authors" title="click to view 80 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '80 more authors' ? 'Alejandra Saori Araki, Ricardo N Araujo, Peter Arensburger, Joshua B Benoit, Reginaldo Pecanha Brazil, Rafaela V Bruno, Gustavo Silva Rivas, Vinicius Abreu, Jason Charamis, Iliano V Coutinho-Abreu, Samara G Costa-LatgÃ©, Alistair Darby, Viv M Dillon, Scott J Emrich, Daniela Fernandez-Medina, Nelder Figueiredo Gontijo, Catherine M Flanley, Derek Gatherer, Fernando A Genta, Sandra Gesing, Gloria I Giraldo-CalderÃ³n, Bruno Gomes, Eric Roberto Guimaraes Rocha Aguiar, James G C Hamilton, Omar Hamarsheh, Mallory Hawksworth, Jacob M Hendershot, Paul V Hickner, Jean-Luc Imler, Panagiotis Ioannidis, Emily C Jennings, Shaden Kamhawi, Charikleia Karageorgiou, Ryan C Kennedy, Andreas Krueger, JosÃ© M Latorre-Estivalis, Petros Ligoxygakis, Antonio Carlos A Meireles-Filho, Patrick Minx, Jose Carlos Miranda, Michael J Montague, Ronald J Nowling, Fabiano Oliveira, JoÃ£o OrtigÃ£o-Farias, Marcio G Pavan, Marcos Horacio Pereira, Andre Nobrega Pitaluga, Roenick Proveti Olmo, Marcelo Ramalho-Ortigao, JosÃ© M C Ribeiro, Andrew J Rosendale, Mauricio R V SantâAnna, Steven E Scherer, NÃ¡gila F C Secundino, Douglas A Shoue, Caroline Silva Moraes, JoÃ£o Silveira Moledo Gesto, Nataly Araujo Souza, Zainulabueddin Syed, Samuel Tadros, Rayane Teles-de-Freitas, Erich L Telleria, Chad Tomlinson, Yara M Traub-CsekÃ¶, JoÃ£o Trindade Marques, Zhijian Tu, Maria F Unger, Jesus Valenzuela, FlÃ¡via V Ferreira, Karla P V Oliveira, Felipe M Vigoder, John Vontas, Lihui Wang, Gareth D Weedall, Elyes Zhioua, Stephen Richards, Wesley C Warren, Robert M Waterhouse, Rod J Dillon, Mary Ann McDowell' : '80 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">80 more authors</span> </div> <div class="periodical"> <em>PLoS Negl. Trop. Dis.</em>, Apr 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1371/journal.pntd.0010862" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pntd.0010862" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Phlebotomine sand flies are of global significance as important vectors of human disease, transmitting bacterial, viral, and protozoan pathogens, including the kinetoplastid parasites of the genus Leishmania, the causative agents of devastating diseases collectively termed leishmaniasis. More than 40 pathogenic Leishmania species are transmitted to humans by approximately 35 sand fly species in 98 countries with hundreds of millions of people at risk around the world. No approved efficacious vaccine exists for leishmaniasis and available therapeutic drugs are either toxic and/or expensive, or the parasites are becoming resistant to the more recently developed drugs. Therefore, sand fly and/or reservoir control are currently the most effective strategies to break transmission. To better understand the biology of sand flies, including the mechanisms involved in their vectorial capacity, insecticide resistance, and population structures we sequenced the genomes of two geographically widespread and important sand fly vector species: Phlebotomus papatasi, a vector of Leishmania parasites that cause cutaneous leishmaniasis, (distributed in Europe, the Middle East and North Africa) and Lutzomyia longipalpis, a vector of Leishmania parasites that cause visceral leishmaniasis (distributed across Central and South America). We categorized and curated genes involved in processes important to their roles as disease vectors, including chemosensation, blood feeding, circadian rhythm, immunity, and detoxification, as well as mobile genetic elements. We also defined gene orthology and observed micro-synteny among the genomes. Finally, we present the genetic diversity and population structure of these species in their respective geographical areas. These genomes will be a foundation on which to base future efforts to prevent vector-borne transmission of Leishmania parasites.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2020</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Karageorgiou2020-kp" class="col-sm-8"> <div class="title">The cyclically seasonal Drosophila subobscura inversion O7 originated from fragile genomic sites and relocated immunity and metabolic genes</div> <div class="author"> <em>Charikleia Karageorgiou</em>, Rosa TarrÄ±Ìo, and Francisco RodrÄ±Ìguez-Trelles </div> <div class="periodical"> <em>Front. Genet.</em>, Oct 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.3389/fgene.2020.565836" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.3389/fgene.2020.565836" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Chromosome inversions are important contributors to standing genetic variation in Drosophila subobscura. Presently, the species is experiencing a rapid replacement of high-latitude by low-latitude inversions associated with global warming. Yet not all low-latitude inversions are correlated with the ongoing warming trend. This is particularly unexpected in the case of O7 because it shows a regular seasonal cycle that peaks in summer and rose with a heatwave. The inconsistent behavior of O7 across components of the ambient temperature suggests that is causally more complex than simply due to temperature alone. In order to understand the dynamics of O7, high-quality genomic data are needed to determine both the breakpoints and the genetic content. To fill this gap, here we generated a PacBio long read-based chromosome-scale genome assembly, from a highly homozygous line made isogenic for an O3 + 4 + 7 chromosome. Then we isolated the complete continuous sequence of O7 by conserved synteny analysis with the available reference genome. Main findings include the following: (i) the assembled O7 inversion stretches 9.936 Mb, containing &gt; 1,000 annotated genes; (ii) O7 had a complex origin, involving multiple breaks associated with non-B DNA-forming motifs, formation of a microinversion, and ectopic repair in trans with the two homologous chromosomes; (iii) the O7 breakpoints carry a pre-inversion record of fragility, including a sequence insertion, and transposition with later inverted duplication of an Attacin immunity gene; and (iv) the O7 inversion relocated the major insulin signaling forkhead box subgroup O (foxo) gene in tight linkage with its antagonistic regulatory partner serine/threonine-protein kinase B (Akt1) and disrupted concerted evolution of the two inverted Attacin duplicates, reattaching them to dFOXO metabolic enhancers. Our findings suggest that O7 exerts antagonistic pleiotropic effects on reproduction and immunity, setting a framework to understand its relationship with climate change. Furthermore, they are relevant for fragility in genome rearrangement evolution and for current views on the contribution of breakage versus repair in shaping inversion-breakpoint junctions.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Karageorgiou2019-wo" class="col-sm-8"> <div class="title">Long-read based assembly and synteny analysis of a reference Drosophila subobscura genome reveals signatures of structural evolution driven by inversions recombination-suppression effects</div> <div class="author"> <em>Charikleia Karageorgiou</em>, VÄ±Ìctor GÃ¡mez-Visairas, Rosa TarrÄ±Ìo, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Francisco RodrÄ±Ìguez-Trelles' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>BMC Genomics</em>, Mar 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1186/s12864-019-5590-8" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1186/s12864-019-5590-8" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>BACKGROUND: Drosophila subobscura has long been a central model in evolutionary genetics. Presently, its use is hindered by the lack of a reference genome. To bridge this gap, here we used PacBio long-read technology, together with the available wealth of genetic marker information, to assemble and annotate a high-quality nuclear and complete mitochondrial genome for the species. With the obtained assembly, we performed the first synteny analysis of genome structure evolution in the subobscura subgroup. RESULTS: We generated a highly-contiguous Â  129 Mb-long nuclear genome, consisting of six pseudochromosomes corresponding to the six chromosomes of a female haploid set, and a complete 15,764 bp-long mitogenome, and provide an account of their numbers and distributions of codifying and repetitive content. All 12 identified paracentric inversion differences in the subobscura subgroup would have originated by chromosomal breakage and repair, with some associated duplications, but no evidence of direct gene disruptions by the breakpoints. Between lineages, inversion fixation rates were 10 times higher in continental D. subobscura than in the two small oceanic-island endemics D. guanche and D. madeirensis. Within D. subobscura, we found contrasting ratios of chromosomal divergence to polymorphism between the A sex chromosome and the autosomes. CONCLUSIONS: We present the first high-quality, long-read sequencing of a D. subobscura genome. Our findings generally support genome structure evolution in this species being driven indirectly, through the inversionsâ recombination-suppression effects in maintaining sets of adaptive alleles together in the face of gene flow. The resources developed will serve to further establish the subobscura subgroup as model for comparative genomics and evolutionary indicator of global change.</p> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> Â© Copyright 2025 Charikleia (Clio) Karageorgiou. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js?a0db7e5d5c70cc3252b3138b0c91dcaf" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?c8a01c11a92744d44b093fc3bda915df" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/assets/js/mathjax-setup.js?a5bb4e6a542c546dd929b24b8b236dfd"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script async src="https://www.googletagmanager.com/gtag/js?id="></script> <script defer src="/assets/js/google-analytics-setup.js"></script> <script defer src="/assets/js/progress-bar.js?2f30e0e6801ea8f5036fa66e1ab0a71a" type="text/javascript"></script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>
    addBackToTop();
  </script> </body> </html>